README.md

```diff
@@ -24,7 +36,42 @@ class Example extends Component {
 }
 ```
 
-## Development
+## Contributing
+
+### What to avoid in RoverUI
+
+It's _not_ a repository for all shared components. Don't try to add components that deal specifically with business logic and concepts here. Don't try to add components that handle whole layout areas.
+
+## Run locally
+
+0. Read the [Code Guidelines](./CODE_GUIDELINES.md)
+1. Download the repository
+1. From the root, run `yarn` to install dependencies.
```

Comment:
you don't wanna update these numbers do you

Reply:
Technically they could be all zeros and I think it would still work

Reply:
I shall update them for prosperity.